### PR TITLE
Fix: Remove default dimensions controls from core/avatar.

### LINKED ISSUE #55590: Remove default dimensions controls from core/avatar
The default padding and margin control is not necessary. We should add this to the block.json: 

```
"__experimentalDefaultControls": {
				"margin": false,
				"padding": false
			}
```
Template

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
Fixes: https://github.com/WordPress/gutenberg/issues/55590

### CODE REVIEW

**Review by cbravobernal — APPROVED** [review]
LGTM - `npm run docs:build` required 😅

**Review by fabiankaegy — CHANGES_REQUESTED** [review]
This change doesn't do what the issue requests. The issue wants that the controls are not shown by default. The options should still be available. 

Removing those would be a larger backwards compatibility issue.

File: packages/block-library/src/avatar/block.json [hunk 1]
```diff
@@ -29,8 +29,8 @@
 		"align": true,
 		"alignWide": false,
 		"spacing": {
-			"margin": true,
-			"padding": true
+			"margin": false,
+			"padding": false
```
Comment (fabiankaegy) [inline packages/block-library/src/avatar/block.json]: What @richtabor suggested in the issue is this here:

```suggestion
			"margin": true,
			"padding": true,
			"__experimentalDefaultControls": {
				"margin": false,
				"padding": false
			}
```

**Review by fabiankaegy — APPROVED** [review]
Looks good to me now 👍

**Pull request merged**